test/vagrant-service-manager/installer_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+require 'vagrant/util/downloader'
+
+module VagrantPlugins
```

Comment:
We are not mocking out all of Vagrant. We still create the `Machine` providing some fake instances. The `Machine` is also responsible for creating the merged Vagrant config. It takes the _Vagrantfile_ in the root directory into account in this case.